doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -147,6 +146,10 @@ New-Item -Path "${DocOutDir}" -Name templates -ItemType directory
 Copy-Item "${DocCommonGenDir}/templates/**" -Destination "${DocOutDir}/templates" -Recurse -Force
 Copy-Item "${DocCommonGenDir}/docfx.json" -Destination "${DocOutDir}" -Force
 
+Write-Verbose "Create Toc for Site Navigation"
+New-Item "${DocOutDir}/toc.yml" -Force
+Add-Content -Path "${DocOutDir}/toc.yml" -Value "- name: ${ArtifactName}`r`n  href: api/`r`n  homepage: index.md"
```

Comment:
Tested in local. We can redirect to root index.html

Reply:
Do you know what the `href: api` does? Should that also be move to empty?

Reply:
Good catch. It is actually searching the toc under api if no homepage initiates. However, just checked the docfx site, and they seems use href instead of homepage right now. 
https://dotnet.github.io/docfx/tutorial/intro_toc.html#link-to-a-folder

I will submit a PR to make the changes like:
```
- name: Azure.Core
  href: index.md
``` 

I believe that's enough